controllers/workspace/devworkspace_controller.go

```diff
@@ -279,7 +294,19 @@ func (r *DevWorkspaceReconciler) Reconcile(req ctrl.Request) (reconcileResult ct
 	deploymentStatus := provision.SyncDeploymentToCluster(workspace, allPodAdditions, serviceAcctName, clusterAPI)
 	if !deploymentStatus.Continue {
 		if deploymentStatus.FailStartup {
-			return r.failWorkspace(workspace, fmt.Sprintf("DevWorkspace spec is invalid: %s", deploymentStatus.Err), reqLogger, &reconcileStatus)
+			var message string
+			if deploymentStatus.Err != nil {
+				message = deploymentStatus.Err.Error()
+				if deploymentStatus.Message != "" {
+					message = message + ": " + deploymentStatus.Message
+				}
+				reqLogger.Info(message)
+			} else {
+				message = deploymentStatus.Message
+				reqLogger.Info(message)
+			}
+
+			return r.failWorkspace(workspace, fmt.Sprintf("DevWorkspace spec is invalid: %s", message), reqLogger, &reconcileStatus)
```

Comment:
I know it's here just from the previous state, but I would say it's not true.
Pod may be failed to start because authentication is missing, or DWO provides the bad init container due DWO config issues or bug, it's not necessary something user may be able to fix with DevWorkspace Spec.
```suggestion
			return r.failWorkspace(workspace, fmt.Sprintf("Startup failed due: %s", message), reqLogger, &reconcileStatus)
```
^ or probably you'll provide even better common wording.

Reply:
Okay, makes sense. I've updated this!